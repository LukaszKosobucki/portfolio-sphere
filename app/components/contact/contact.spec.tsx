import { formatSnapshot } from "@/app/utils/tests/format-snapshot";
import { mockSocials } from "@/app/utils/tests/mocks/socials.mock";
import { render } from "@testing-library/react";
import { describe, expect, test } from "bun:test";
import Contact from "./contact";

describe("Contact Component", () => {
  test("Should match snapshot", () => {
    const { asFragment } = render(<Contact socials={mockSocials} />);

    expect(formatSnapshot(asFragment())).toMatchSnapshot();
  });
});
