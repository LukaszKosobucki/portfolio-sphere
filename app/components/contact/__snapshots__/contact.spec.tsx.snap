// Bun Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Component Should match snapshot 1`] = `
"<DocumentFragment>
  <section
    class="animate"
  >
    <div>
      <p
        class="font-semibold text-black dark:text-white"
      >
        Let's Connect
      </p>
      <p>
        Reach out to me via email or on social media.
      </p>
      <div
        class="grid grid-rows-3 gap-y-2 mt-4 auto-cols-min"
      >
        <section
          class="flex flex-row gap-2"
        >
          <div
            class="col-span-1 flex items-center gap-1"
          >
            <span
              class="whitespace-nowrap truncate"
            >
              Email
            </span>
          </div>
          <div
            class="col-span-3 truncate"
          >
            <a
              class="w-fit col-span-3 group flex gap-1 items-center underline decoration-[.5px] decoration-black/25 dark:decoration-white/50 hover:decoration-black dark:hover:decoration-white text-black dark:text-white underline-offset-2 blend"
              href="mailto:business@lukaszkosobucki.pl"
              target="_blank"
            >
              <span
                class="text-black/75 dark:text-white/75 group-hover:text-black group-hover:dark:text-white blend"
              >
                business@lukaszkosobucki.pl
              </span>
            </a>
          </div>
        </section>
        <section
          class="flex flex-row gap-2"
        >
          <div
            class="col-span-1 flex items-center gap-1"
          >
            <span
              class="whitespace-nowrap truncate"
            >
              Github
            </span>
          </div>
          <div
            class="col-span-3 truncate"
          >
            <a
              class="w-fit col-span-3 group flex gap-1 items-center underline decoration-[.5px] decoration-black/25 dark:decoration-white/50 hover:decoration-black dark:hover:decoration-white text-black dark:text-white underline-offset-2 blend"
              href="https://github.com/LukaszKosobucki"
              target="_blank"
            >
              <span
                class="text-black/75 dark:text-white/75 group-hover:text-black group-hover:dark:text-white blend"
              >
                LukaszKosobucki
              </span>
            </a>
          </div>
        </section>
        <section
          class="flex flex-row gap-2"
        >
          <div
            class="col-span-1 flex items-center gap-1"
          >
            <span
              class="whitespace-nowrap truncate"
            >
              LinkedIn
            </span>
          </div>
          <div
            class="col-span-3 truncate"
          >
            <a
              class="w-fit col-span-3 group flex gap-1 items-center underline decoration-[.5px] decoration-black/25 dark:decoration-white/50 hover:decoration-black dark:hover:decoration-white text-black dark:text-white underline-offset-2 blend"
              href="https://www.linkedin.com/in/lukasz-kosobucki/"
              target="_blank"
            >
              <span
                class="text-black/75 dark:text-white/75 group-hover:text-black group-hover:dark:text-white blend"
              >
                lukasz-kosobucki
              </span>
            </a>
          </div>
        </section>
      </div>
    </div>
  </section>
</DocumentFragment>"
`;
